.github/workflows/flatpak.yml

```diff
@@ -0,0 +1,171 @@
+name: Flatpak
+
+on:
+  release:
+    types: push
```

Comment:
Is this documented somewhere? I don't see `push` here, but maybe I'm looking at the wrong place or missing something.

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release

Reply:
Now resolved?

Reply:
OK, so is it better to push before other prs now or at the final (to not create a lot of workflows)?

Reply:
Probably best to merge this last.